consensus/replay.go

```diff
@@ -264,10 +264,12 @@ func (h *Handshaker) ReplayBlocks(state sm.State, appHash []byte, appBlockHeight
 			Validators:    validators,
 			AppStateBytes: h.appState,
 		}
-		_, err := proxyApp.Consensus().InitChainSync(req)
+		res, err := proxyApp.Consensus().InitChainSync(req)
 		if err != nil {
 			return nil, err
 		}
+		state.Validators = res.Validators
+		return res.AppHash, nil
```

Comment:
we still want to run the replay even if we're sending InitChain - the blockchain might have 1000 blocks in it even though the app wants to start from 0 ...